Sources/NIO/Linux.swift

```diff
@@ -113,6 +113,10 @@ internal enum Epoll {
 }
 
 internal enum Linux {
+    static let CFS_QUOTA_PATH = "/sys/fs/cgroup/cpu/cpu.cfs_quota_us"
+    static let CFS_PERIOD_PATH = "/sys/fs/cgroup/cpu/cpu.cfs_period_us"
+    static let CPUSET_PATH = "/sys/fs/cgroup/cpuset/cpuset.cpus"
+    struct LinuxError: Error { }
```

Comment:
Pretty much everything that's a `LinuxError` should never happen so we can remove this error I think.